src/cfg.rs

```diff
@@ -18,28 +19,59 @@ use serde::Deserialize;
 #[derive(Debug, Deserialize)]
 pub struct Cfg {
     pub layers: Layers,
+    pub layer_aliases: HashMap<usize, Option<String>>,
+    pub tap_hold_wait_time: u64,
+    pub tap_dance_wait_time: u64,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct PreCfg {
```

Comment:
Wondering if we could implement this without the `PreCfg`.
I think that we can just keep the non-exaustive `layer_aliases` map, and pass it to the `LayersManager`.

Instead of storing a `None` for each non-aliased layer index, we can conclude from the fact that it's missing from the aliases map that it doesn't have an alias.
WDYT?

Reply:
Sure. Let me see what I can do on that front.

Reply:
see latest